draft-irtf-cfrg-frost.md

```diff
@@ -472,12 +492,12 @@ on the signer commitment list and message to be signed.
 
   def compute_binding_factors(commitment_list, msg):
     msg_hash = H3(msg)
+    encoded_commitment_hash = H3(encode_group_commitment_list(commitment_list))
```

Comment:
Could this be `H3(msg || encode_group_commitment_list(commitment_list))` ? // @chelseakomlo

Reply:
Perhaps not because chosen message something something